fluent-syntax/src/parser.js

```diff
@@ -179,7 +226,7 @@ export default class FluentParser {
     ps.expectChar(']');
     ps.expectChar(']');
 
-    return new AST.Section(symb, comment);
+    return new AST.Section(symb);
```

Comment:
`getSection` should return a `GroupComment` with the section's comment or `null` (or `undefined`) if the comment is missing.